### PR TITLE
Added more SLES Support

### PR DESCRIPTION
**Pull request by eradot4027**
#### Description:

- Added support for SUSE Linux in checks to support CIS Draft Benchmark
#### Rationale:

- Support for SSH packages and FIPS are required checks in the benchmark

### CODE REVIEW

File: linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_FIPS_certified/rule.yml [hunk 1]
```diff
@@ -11,6 +11,10 @@ description: |-
 {{% if product in ["rhel6", "rhel7"] %}}
     Red Hat Enterprise Linux is supported by Red Hat, Inc. As the Red Hat Enterprise
     Linux vendor, Red Hat, Inc. is responsible for maintaining government certifications and standards.
+{{% elif product in ["sle12"] %}}
+    SUSE Enterprise Linux is supported by SUSE Software Solutions Germanu GmbH. As the SUSE Enterprise
+    Linux vendor, SUSE Software Solutions Germanu GmbH is responsible for maintaining government
+    certifications and standards.
```
Comment (shawndwells) [inline linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_FIPS_certified/rule.yml:17]:
```suggestion
{{% elif product in ["sle12", "sle15"] %}}
    SUSE Enterprise Linux is supported by SUSE Software Solutions Germanu GmbH. As the SUSE Enterprise
    Linux vendor, SUSE Software Solutions Germanu GmbH is responsible for maintaining government
    certifications and standards.
```

Comment (shawndwells) [reply linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_FIPS_certified/rule.yml:17]: Should there be a similar text for ``sle15``?

Comment (eradot4027) [reply linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_FIPS_certified/rule.yml:17]: My reasoning  behind leaving it off was that SLE15 has not yet achieved FIPS certification (according to their website).  I reasoned that is why there was no entry for RHEL8 in that same block.  If that is mistaken, I will gladly add it in.

Comment (shawndwells) [reply linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_FIPS_certified/rule.yml:17]: @eradot4027 Makes sense! Your reasoning is spot on. I don't track SLE15 so unaware of what FIPS has been achieved, so appreciate your keeping everyone accurate on this.

File: linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_FIPS_certified/rule.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: fedora,ocp4,ol7,ol8,rhel6,rhel7,rhel8,rhv4
+prodtype: fedora,ocp4,ol7,ol8,rhel6,rhel7,rhel8,rhv4,sle12,sle15
```
Comment (shawndwells) [inline linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_FIPS_certified/rule.yml:3]: To make specific mention: SLE15 is on the NIST Modules in Process list (https://csrc.nist.gov/Projects/cryptographic-module-validation-program/Modules-In-Process/Modules-In-Process-List) and Implementation Under Test list (https://csrc.nist.gov/Projects/cryptographic-module-validation-program/Modules-In-Process/IUT-List)

For that reason, adding SLE15 seems appropriate. Was the same gate RHEL8 was held against.

Comment (eradot4027) [reply linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_FIPS_certified/rule.yml:3]: I have it in the list, but removed it from the check phase as it is still in process.  I at least want it to return a negative until it is certified

Comment (shawndwells) [reply linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_FIPS_certified/rule.yml:3]: Totally your call. RHEL8 was only added after being formally listed as in progress by NIST. 

We should hold other distros to the same standard/gate. If you feel its better to hold off, no worries. I will also merge if you want to add it now.

Comment (eradot4027) [reply linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_FIPS_certified/rule.yml:3]: I'll add it and push.  That way we don't have to waste review cycles in the future on such a small changes

**Comment by shawndwells** [comment]
@openscap-ci test this please

**Pull request merged**